a.txt

```diff
@@ -0,0 +1,61 @@
+stems=/nfs/sc/proj/icl/devshare/kbobrovs/drivers/neo/install/igc|/rdrive/ref/opencl/runtime/linux/oclgpu
```

Comment:
do we really need this file? Please remove it or at east remove all usernames and local/nonstandard paths

Reply:
We don't. This file was removed in one of the subsequent pushes